src/common.js

```diff
@@ -367,11 +369,13 @@ function wrap(cmd, fn, options) {
           if (e.msg === 'earlyExit') {
             retValue = e.retValue;
           } else {
+            /* istanbul ignore next */
```

Comment:
With `ignore next`, I think istanbul will mark the `else` route as not taken. `ignore else` should improve test coverage slightly.